core/src/main/java/com/eternalcode/core/feature/essentials/mob/MobType.java

```diff
@@ -0,0 +1,59 @@
+package com.eternalcode.core.feature.essentials.mob;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.bukkit.entity.Animals;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.Monster;
+import org.bukkit.entity.Mob;
+import org.jetbrains.annotations.Nullable;
+
+public enum MobType {
+    PASSIVE(Animals.class, true, true),
+    AGGRESSIVE(Monster.class, true, true),
+    OTHER(false, false),
+    UNDEFINED(false, false),
+    ALL(true, true);
+
+    @Getter @Setter
```

Comment:
We only use Lombok in plugin configuration classes, please create a getter and setter and remove Lombok from this class